packages/js-moi-providers/__tests__/jsonrpc-provider.test.ts

```diff
@@ -169,27 +189,51 @@ describe("Test JsonRpcProvider Query Calls", () => {
     describe('getTesseract', () => {
         it('should return the latest tesseract', async () => {
           const tesseract = await provider.getTesseract(address, false);
+          const participant = tesseract.participants.find(p => p.address === address);
+
+          if (!participant) {
+            expect(participant).toBeDefined();
+            return;
+          }
```

Comment:
The if statement can be avoided

Reply:
ff1526080cac6383d6521a80b129d8a610dc7f99